web/src/components/hooks/useIsHelmManaged.js

```diff
@@ -1,39 +0,0 @@
-import { useQuery } from "react-query";
-import { Utilities } from "../../utilities/utilities";
-
-async function fetchIsHelmManaged({
-  accessToken = Utilities.getToken(),
-  apiEndpoint = process.env.API_ENDPOINT,
-  _fetch = fetch,
-} = {}) {
-  try {
-    const res = await _fetch(`${apiEndpoint}/is-helm-managed`, {
-      headers: {
-        Authorization: accessToken,
-        "Content-Type": "application/json",
-      },
-      method: "GET",
-    });
-    if (res.ok) {
-      const response = await res.json();
-      return { isHelmManaged: response.isHelmManaged };
```

Comment:
decided not to do this kind of transformation in the fetch handler. this is technically a mutation that we should unit test and we can do it in the `select`